runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/EngineBuilder.java

```diff
@@ -39,10 +39,12 @@ public class EngineBuilder
     private Configuration config;
     private ErrorHandler errorHandler;
     private Collection<EngineAffinity> affinities;
+    private boolean readonly;
 
     EngineBuilder()
     {
         this.affinities = new LinkedHashSet<>();
+        this.readonly = false;
```

Comment:
`boolean` already defaults to `false` implicitly per Java Language Specification, which is why we pick the polarity of `readonly` instead of `writeable` for the boolean field, so no need to re-assign to `false` in the constructor.